src/dashboards/components/DashboardPage.tsx

```diff
@@ -145,9 +150,12 @@ const mstp = (state: AppState) => {
     state.currentDashboard.id
   )
 
+  const showAnnotationBar = state.userSettings.showAnnotationsControls || false
```

Comment:
```state.userSettings.showAnnotationsControls || false```

i like where your head's at here, but i think this is one of those situations that calls for failing fast, because if this doesn't have a default value, it means our state isn't set up correctly. in redux, you give default values to every part of your state when you create it. that way, in component land, we can safely assume that this thing will have a default value. i appreciate you checking it, that's good practice, but that check should already be happening as part of the `userSettings` reducer `initialState` method.